appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: 1.0.{build}
+
+environment:
+  PATH: C:\sbt\bin;%PATH%
```

Comment:
The directory for extracting the zip is specified in appveyor.ps1 script line number 13. This absolute path is just a reference to that.